src/controllers/UserController.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Italia PagoPA Proxy
+ * Cittadinanza Digitale PagoPA services
+ *
+ */
+
+import * as Mocked from "../MockedData";
+
+import { none, Option, some } from "fp-ts/lib/Option";
+
+import { CreditCard } from "../types/CreditCard";
+import { PaymentMethod, PaymentMethodType } from "../types/PaymentMethod";
+import { ITransaction } from "../types/Transaction";
+import { IUser } from "../types/User";
+
+/**
+ * User Controller
+ */
+export class UserController {
```

Comment:
I see no state here, this controller can be a expressed with simple functions (instead of a Class). when you need
encapsulation (namespacing) use javascript modules not classes.